bridge/src/lib.rs

```diff
@@ -170,6 +170,13 @@ pub mod pallet {
 		RegisterDestDomain { sender: T::AccountId, domain_id: DomainID, chain_id: ChainID },
 		/// When unregistering a dest domainID with its corresponding chainID
 		UnregisterDestDomain { sender: T::AccountId, domain_id: DomainID, chain_id: ChainID },
+		/// When bridge fee is collected
+		FeeCollected {
+			fee_payer: T::AccountId,
+			dest_domain_id: DomainID,
+			resource_id: ResourceId,
+			fee_amount: u128,
```

Comment:
that makes a lot sense, but our relayer side dev has already implemented the parsing feeCollected event, so I will just add AssetID field instead of changing fee_amount to fee: MultiAsset